magic_bridge/ic/src/magic_bridge/src/api/dab.rs

```diff
@@ -0,0 +1,22 @@
+use ic_kit::Principal;
+use ic_kit::{macros::update};
+
+use crate::factory::CreateCanisterParam;
+use crate::{
+    magic::STATE, 
+    dab::retry_failed_canisters,
+};
+
+use crate::api::admin::is_authorized;
+
+#[update(name = "flush_failed_registrations", guard = "is_authorized")]
```

Comment:
```suggestion
#[update(name = "flush_failed_registrations", guard = "is_authorized")]
#[candid_method(update, rename = "flush_failed_registrations")]
```
allows us to export method definitions